lib/public/views/Runs/ActiveColumns/getQcSummaryDisplay.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h, iconWarning, iconBolt } from '/js/src/index.js';
+import { formatFloat } from '../../../utilities/formatting/formatFloat.js';
+import { getValueFromCssVar, isContrastBlack } from '../../../components/common/badge.js';
+import { QC_SUMMARY_COLORS } from '../../../components/qcFlags/qcSummaryColors.js';
+import { tooltip } from '../../../components/common/popover/tooltip.js';
+
+const FULL_COVERAGE = 1;
+
+/**
+ * Render display common for QC and GAQ summaries
+ *
+ * @param {RunDetectorQcSummary|RunGaqSummary} summary summary
+ * @return {Component} display
+ */
+export const getQcSummaryDisplay = (summary) => {
+    const {
+        badEffectiveRunCoverage,
+        explicitlyNotBadEffectiveRunCoverage,
+        missingVerificationsCount,
+        mcReproducible,
+    } = summary;
+
+    const missingVerificationMessage = `Missing ${missingVerificationsCount} verification${missingVerificationsCount > 1 ? 's' : ''}`;
+    const missingVerificationDisplay = missingVerificationsCount
+        ? h(
+            '.d-inline-block.va-t-bottom',
+            tooltip(h('.f7', iconWarning()), missingVerificationMessage),
+        )
+        : null;
+
+    const notBadPercentageFormat = h('span', formatFloat((1 - badEffectiveRunCoverage) * 100, { precision: 0 }));
+
+    const nonMcReproducibleQcSummaryContent = [
+        notBadPercentageFormat,
+        missingVerificationDisplay,
+    ];
+
+    /**
+     * Format QC summary with color
+     * @param {string|Component} content content
+     * @param {string} [configuration.color] the color to look for contrast,
+     * in hexa RGB or CSS var() function over variable name expression: e.g. "var(--color-nice)"
+     * @returns {Component} QC summary display
```

Comment:
You are using `return` above and `returns` here.
Please keep to `return` usage

Reply:
applied